src/pages/modals/trackModals/trackModalHelpers.js

```diff
@@ -0,0 +1,48 @@
+import { changeDateFormat, isObjectFieldsEmpty } from '../../../scripts/helpers';
+import { TRACK_MODAL_TITLES } from '../../../scripts/libraries';
+
+export const initTrackModalState = {
+  modalTitle: '',
+  taskName: '',
+  note: '',
+  date: '',
+  readOnly: false,
+  formErrors: {
+    note: '',
+    date: '',
+  },
+};
+
+export const getTrackModalErrors = (note, date) => {
+  const formErrors = {
+    note: '',
+    date: '',
+  };
+  if (!note.trim()) {
+    formErrors.note = 'Note is required';
+  }
+  if (!date.trim()) {
+    formErrors.date = 'Date is required';
+  }
```

Comment:
You have a lot of similar logic in your helpers. Let's merge it.

Reply:
Yeap, this function is like `getTaskModalErrors`. Replased to this 
in both functions:
```suggestion
  const keys = Object.keys(formErrors);
  keys.forEach((key) => {
    if (!state[key].trim()) {
      formErrors[key] = `${MODAL_VALUES[key]} is required`;
    }
  });
```